examples/20-secret-cloudflare-credentials.yaml

```diff
@@ -8,4 +8,5 @@ data:
   # replace '...' with the API Key generated. The token should have permission of Zone:Read and DNS:Edit for all zones.
   # You can optionally exclude certain zones
   # see https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys
+  # For details see https://github.com/SayakMukhopadhyay/external-dns-management/blob/master/doc/cloudflare/README.md#using-the-api-key
```

Comment:
Please adjust the URLs `https://github.com/SayakMukhopadhyay/external-dns-management/...`

Reply:
Fixed the issue.